.changeset/config.json

```diff
@@ -7,6 +7,7 @@
     }
   ],
   "ignore": [
+    "@contentful/f36-icon-alpha",
```

Comment:
Remnant of publishing the wrong package during some testing. To be removed and [published package](https://www.npmjs.com/package/@contentful/f36-icon-alpha) to be unpublished.